client/src/services/SettingsService.js

```diff
@@ -102,6 +104,23 @@ class SettingsService {
     this.save();
   }
 
+  setModelsColors(modelIds) {
+    this.modelsColors = [];
+    for (let i = 0; i < modelIds.length; i++) {
+      const im = i % colors.length;
+      this.modelsColors[modelIds[i]] = `#${(colors[(colors.length - 1) - im])}`;
+    }
+  }
+
+  getModelColor(modelId) {
+    if (this.modelsColors[modelId]) {
+      return this.modelsColors[modelId];
+    }
+    // If the list of colors is not set yet return a random from the list
```

Comment:
In what circumstance would this occur?

Reply:
Probably never; it was just in case, removed